src/style/style_layer.js

```diff
@@ -193,7 +194,10 @@ class StyleLayer extends Evented {
         return false;
     }
 
-    updatePaintTransitions(options: any, globalOptions: any, animationLoop: any, zoomHistory: any) {
+    updatePaintTransitions(options: {transition?: boolean},
+                           globalOptions?: TransitionSpecification,
+                           animationLoop?: AnimationLoop,
+                           zoomHistory?: any) {
```

Comment:
is zoomHistory here still `any` because of the `ZoomHistory | {}` in Style?

Reply:
Yeah... we should fix that eventually.